consensus/ninja_tortoise.go

```diff
@@ -64,7 +64,7 @@ func (vp votingPattern) Layer() mesh.LayerID {
 type BlockCache interface {
 	GetBlock(id mesh.BlockID) (*mesh.Block, error)
 	LayerBlockIds(id mesh.LayerID) ([]mesh.BlockID, error)
-	ForBlockInView(view map[mesh.BlockID]struct{}, layer mesh.LayerID, foo func(block *mesh.BlockHeader), errHandler func(err error))
+	ForBlockInView(view map[mesh.BlockID]struct{}, layer mesh.LayerID, foo mesh.TraversalFunc, errHandler func(err error)) error
```

Comment:
i prefer it stayed func(block *mesh.BlockHeader) or whatever 
it dosen't have to be related to mesh